### ISSUE TITLE
sitewide navigation does not meet WCAG contrast guidelines

### ISSUE THREAD

**Issue by ghwood** [ietf-tools/datatracker#6112]
### Describe the issue

it looks like that at some point the datatracker.ietf.org nav bar contrast changed and now doesnt meet WCAG guidelines for contrast.  Here is the issue I see reported:

"Element has insufficient color contrast of 4.17 (foreground color: #0b0c0c, background color: #6c757d, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

Related Node
<nav class="navbar navbar-expand-lg  bg-secondary    fixed-top ">
"

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by larseggert** [comment]
Out of curiosity, how did you get this WCAG warning? What service did you use?

**Comment by ghwood** [comment]
I used the [aXe extension](https://github.com/dequelabs/axe) for Chrome
